tasks/ci.rake

```diff
@@ -193,11 +204,13 @@ EOS
     task :gem => ['ci:check_env'] do
       beaker(:init, '--hosts', 'config/nodes/gem.yaml', '--options-file', 'config/gem/options.rb')
       beaker(:provision)
-      beaker(:exec, 'pre-suite', '--pre-suite', pre_suites(:gem))
-      beaker(:exec, "#{File.dirname(__dir__)}/setup/gem/010_GemInstall.rb")
-
-      preserve_hosts = ENV['OPTIONS'].include?('--preserve-hosts=always') if ENV['OPTIONS']
-      beaker(:destroy) unless preserve_hosts
+      begin
```

Comment:
this is to make handling gem processing consistent with other c:test options, i.e., always process the destroy regardless of upstream failures.